SimplePolicy/SimplePolicyPlugin.cs

```diff
@@ -63,7 +66,7 @@ private static IEnumerable<Transaction> FilterForBlock_Policy2(IEnumerable<Trans
             Transaction[] free = tx_list.Where(p => p.IsLowPriority)
                 .OrderByDescending(p => p.NetworkFee / p.Size)
                 .ThenByDescending(p => p.NetworkFee)
-                .ThenByDescending(p => InHighPriorityList(p))
+                .ThenByDescending(InHighPriorityList)
```

Comment:
I think that was more readable before this line change

Reply:
Seems like just extra characters to me, but I can put it back if others think it is more readable.

Reply:
No one else commented, but I put it back for you @shargon :)